secboot/secboot_tpm.go

```diff
@@ -28,6 +28,7 @@ import (
 	"os"
 
 	"github.com/canonical/go-tpm2"
+	"github.com/snapcore/secboot"
 	sb "github.com/snapcore/secboot"
```

Comment:
why is snapcore/secboot imported for a second time?
Is `sb "github.com/snapcore/secboot"` somehow insufficient?

Reply:
this must have been a mistake and I can't see this on master so I think this was eventually corrected

Reply:
@anonymouse64 I still see it here https://github.com/snapcore/snapd/blob/master/secboot/secboot_tpm.go#L31 is this the wrong place?

Reply:
Thanks for spotting. I'll open a PR

Reply:
and it's up https://github.com/snapcore/snapd/pull/10655